### PR TITLE
Readable mint error

### PR DESCRIPTION
**Pull request by hairtail**
## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish/src/primitives/rawTransaction.test.ts [hunk 1]
```diff
@@ -170,7 +178,9 @@ describe('RawTransaction', () => {
     })
 
     expect(() => raw.post(account.spendingKey)).toThrow(
-      'Cannot post transaction. Burn value exceededs maximum',
+      `Cannot post transaction. Burn value ${CurrencyUtils.renderIron(
+        burn.value,
+      )} exceededs maximum ${CurrencyUtils.renderIron(MAX_MINT_OR_BURN_VALUE)}`,
```
Comment (NullSoldier) [inline ironfish/src/primitives/rawTransaction.test.ts:183]: Don't change these errors in the test, you can do partial checking of an error using `toThrowError`

Comment (hairtail) [reply ironfish/src/primitives/rawTransaction.test.ts]: Try to match error with regex in the latest commit.

**Comment by NullSoldier** [comment]
@hairtail What graffiti do you want this credited to?

**Comment by hairtail** [comment]
> @hairtail What graffiti do you want this credited to?

https://testnet.ironfish.network/users/14586

**Pull request merged**